lib/prototypes.h

```diff
@@ -24,6 +24,7 @@
 #include <sys/socket.h>
 #include <sys/stat.h>
 #include <sys/types.h>
+#include <unistd.h>
```

Comment:
The include for `pid_t` is `<sys/types.h>`.  See pid_t(3types).

<https://www.man7.org/linux//man-pages/man3/pid_t.3type.html>

Reply:
Good point, thanks.